src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -238,6 +237,7 @@ protected virtual async Task ExecuteNodeAsync(ExecutionContext context, Executio
                 {
                     CompleteNode(context, node);
                     // for non-dataloader nodes that completed without throwing an error, we can re-use the context
+                    resolveContext.Reset(null, null);
```

Comment:
Also note that we need clear resolveContext before returning it to the "pool". Otherwize, we extend the lifetime of its content indefinitely.

Reply:
Well, through the lifetime of the execution context anyway.  I already did this in FuncFieldResolver because that can actually be indefinitely.

Reply:
That is, do we leave this cleanup line here?